root/static/scripts/common/components/Autocomplete2/formatters.js

```diff
@@ -117,7 +119,23 @@ function formatArtist(artist: ArtistT) {
       info.unshift(sortName);
     };
   }
-  return formatGeneric(artist, extraInfo);
+
+  if (nonEmpty(artist.typeName)) {
+    secondInfoRow.push(lp_attributes(artist.typeName, 'artist_type'));
+  }
+
+  if (artist.begin_date || artist.end_date) {
```

Comment:
I'd recommend `isDateEmpty` here in case we get a partial date object with all nulls somewhere